examples/ProgressDialog.py

```diff
@@ -9,7 +9,7 @@
 import pyqtgraph as pg
 from pyqtgraph.Qt import QtCore, QtGui
 
-app = QtGui.QApplication([])
+app = pg.mkQApp("Process Dialog Example")
```

Comment:
s/Process/Progress/g

Reply:
thank you!

Reply:
I am not sure about this calling of mkQApp() with an application name parameter. Where does it get shown?

Reply:
This PR might actually be the first usage of that feature, but it was introduced in #1166 and further discussed in #1199 (don't judge the PR numbers - those are really the right ones!).

Reply:
with name parameter: 
![image](https://user-images.githubusercontent.com/646398/105796838-8e046680-5f44-11eb-9acf-ca996b4b3b2c.png)

without name parameter:
![image](https://user-images.githubusercontent.com/646398/105796867-9c528280-5f44-11eb-8338-dacae383369b.png)